src/components/TextInput/InputControl/InputControl.tsx

```diff
@@ -17,6 +19,8 @@ export function InputControl(props: InputControlProps) {
         autoFocus,
         autoComplete,
         placeholder,
+        innerLabel,
+        innerLabelWidth,
```

Comment:
Can we do it automatically?

Reply:
yes, moved setting paddingLeft to parent component (TextInput). no need to add extra props here.